math/src/elliptic_curve/short_weierstrass/curves/bls12_381/field_extension.rs

```diff
@@ -244,6 +275,16 @@ mod tests {
     use super::*;
     type Fp12E = FieldElement<Degree12ExtensionField>;
 
+    #[test]
+    fn conjugate_works() {
+        let a: QuadraticExtensionFieldElement<MyQuadraticNonResidue> =
+            QuadraticExtensionFieldElement::zero();
+        let mut expected = a.conjugate();
+        expected = expected.conjugate();
+
+        assert_eq!(a, expected);
+    }
```

Comment:
This test is not testing that conjugate is properly implemented. It only tests that its square is the identity, which is also true for the identity function. I mean. `conjugate` could be implemented as the identity and this test would pass. 
But besides that. Why are you adding this test? This is functionality of generic quadratic extension fields and its already tested [here](https://github.com/lambdaclass/lambdaworks/blob/314fafcf44e1764dafa748f82d6df045f309cbc6/math/src/field/extensions/quadratic.rs#L250-L255).
Please remove it and also remove `MyQuadraticNonResidue` which is only used for this I guess.